### DISCUSSION TITLE
Error deleting message, API responded with status 403!	{"message":"Missing Access","code":50001}

### DISCUSSION THREAD

**Discussion by tawtis** [victornpb/undiscord#482] [1. Questions & Answers]
Any way to skip a certain channel? Clearly I have left this server and can't access my messages in there.

**Comment by wwincast** [comment]
Had the same issue just right now, it could have been your auth token.
All i had to do was click fill to correct the issue.
<img width="250" alt="Screenshot 2023-03-31 at 5 15 36 PM" src="https://user-images.githubusercontent.com/117790693/229255692-301ac8b6-7281-4aa5-b912-49685e009174.png">

**Comment by 573rn** [comment]
If the 403 error is because of a bot:

In Regex enter this:

`^(?!\s*$)(?!.*\bwarn\b).*$`

This works specifically with a GPT bot on the server I'm in.

(?!\s*$) = Whitespace: the bot in my server often posts posts that are only whitespace, aka: no characters at all. This skips posts with no input.

(?!.*\bwarn\b) = The other post the bot makes contains the word "Warn", this skips all posts that contain the word "Warn"

Replace these with whatever unique words or phrases the bot in the server you are trying to avoid uses.

**Reply by linksafe121** [reply]
Hey How do I do this? I can't seem to follow. I use ViolentMonkey

**Reply by ghost** [reply]
I'm still having this issue, did you find a solution?

**Comment by NicholasDewberryOfficial** [comment]
Im still having this issue in version 5.2.3

I've tried it in chrome and chrome incognito. I think I only have this issue if I use the index json file. doing it manually is fine - I think?

**Comment by ghost** [comment]
still an issue, only happens when using index.json
